gossip/src/crds_gossip_pull.rs

```diff
@@ -50,8 +50,6 @@ use {
 };
 
 pub const CRDS_GOSSIP_PULL_CRDS_TIMEOUT_MS: u64 = 15000;
-// The maximum age of a value received over pull responses
-pub const CRDS_GOSSIP_PULL_MSG_TIMEOUT_MS: u64 = 60000;
```

Comment:
can't imagine this is being used externally but, being as it's part of a public interface, consider deprecating rather than outright removal